RadixWallet/Core/DesignSystem/Components/Hint.swift

```diff
@@ -64,29 +64,40 @@ public struct Hint: View, Equatable {
 				}
 				text
 					.lineSpacing(0)
-					.textStyle(.body2HighImportance)
+					.textStyle(textStyle)
 			}
 			.foregroundColor(foregroundColor)
 		}
 	}
+}
 
-	private var foregroundColor: Color {
+private extension Hint {
+	var foregroundColor: Color {
 		switch viewState.kind {
 		case .info:
 			.app.gray2
 		case .error:
 			.app.red1
-		case .warning:
+		case .warning, .detail:
 			.app.alert
 		}
 	}
 
-	private var iconResource: ImageResource? {
+	var iconResource: ImageResource? {
 		switch viewState.kind {
-		case .info:
+		case .info, .detail:
 			nil
 		case .error, .warning:
 			.error
 		}
 	}
+
+	var textStyle: TextStyle {
+		switch viewState.kind {
+		case .info, .error, .warning:
+			.body2HighImportance
```

Comment:
Based on Zeplin it looks like there are situations where `info` and `error` should be `body2Regular` 🤔

Reply:
[info1](https://zpl.io/ykQnLZG), [info2](https://zpl.io/KGWRWlA), [error1](https://zpl.io/B1lXM4K), [error2](https://zpl.io/1MJ33qE)

> after confirming with Aftab that we want them differently

right, maybe they should be consistent across all screens